pom.xml

```diff
@@ -134,11 +135,21 @@
             </excludes>
           </configuration>
         </plugin>
+        <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-jar-plugin</artifactId>
+          <version>3.2.0</version>
```

Comment:
Shall we move this version to a property too?

Reply:
Yep, good candidate for a help-wanted issue.

Reply:
OK, for some reason I thought these plugins were already there (I didn't even create the CQs for these).
We definitely need to set these as properties form the start.

Reply:
ok, properties defined in da100d7